raw-server/server.js

```diff
@@ -0,0 +1,47 @@
+require('dotenv').config();
+const express = require('express');
+const axios = require('axios');
+
+const app = express();
+const port = process.env.RAW_SERVER_PORT || 3002;
+const serverHost = process.env.SERVER_HOST;
```

Comment:
Same thing about process.env usage - should we reference it in common.config.js, and then import from common config (in server part)?

Reply:
This server consists of the single file, so don't think it's so critical in this situation.

Reply:
Probably yes, it's small then :)